dockerfiles/remote-plugin-php7/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia-endpoint-runtime:${BUILD_TAG}
+
+RUN apk --update --no-cache add \
+    ca-certificates
+
+RUN apk --update --no-cache add \
+    php7 php7-fpm php7-opcache
+
+RUN apk --update --no-cache add \
+    php7-gd php7-mysqli php7-zlib php7-curl
+
+RUN apk --update --no-cache \
```

Comment:
we could merge several RUN instructions to decrease the layers

Reply:
I created PR for fixing this comment. https://github.com/vrubezhny/che-theia/pull/1

Reply:
@monaka I have already provided a fixup that merges all `RUN apk...` commands into a single `RUN` command. So now it turns out that your PR conflicts a bit with my fixup. 
Could you please rebase your PR over the fixup commit https://github.com/eclipse/che-theia/pull/203/commits/9a07022bf2f9840e723fb9dca8b5094565aff78b ? 
(Or, as I'll have to squash the commits in the PR before we merge it, can I use (read as `manuallly apply`) your commit when squashing the commits in order to make it all to be a single `RUN` command?)

Reply:
vrubezhny#1 is just for your convenience. Please ignore that and re-push your additional fixes to here.

Reply:
Thanks for your additional fix. LGTM.